test/tests/functional/pbs_multi_sched.py

```diff
@@ -2084,6 +2089,7 @@ def test_multi_sched_job_sort_key(self):
         self.server.expect(JOB, {'job_state': 'R'}, id=jid2)
         self.server.expect(JOB, {'job_state': 'Q'}, id=jid1)
 
+    @skipOnCpuSet
```

Comment:
Add decorator to test cases which are using self.setup_queues_nodes and self.common_setup